be/src/vec/aggregate_functions/aggregate_function_regr_slope.h

```diff
@@ -0,0 +1,200 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
better union the slope and intercept the code，seems only one struct and use template can do the work

Reply:
ok, I will union them later. but there is a quesion that what should I name the file that union the slope and intercept function.